src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -536,28 +540,35 @@ def _describe(self, catalog, database, table) -> Table | None:
         catalog and database.
         """
         full_name = f"{catalog}.{database}.{table}"
```

Comment:
Do we need add the backticks here, i.e. safe quoting?

Reply:
This is only used for logging.